api/api.go

```diff
@@ -66,7 +66,7 @@ func Setup(ctx context.Context, cfg *config.Config, router *mux.Router, dataStor
 }
 
 // enablePublicEndpoints register only the public GET endpoints.
```

Comment:
Rather than blanking these, might be worth removing them as parameters and updating their reference functions.

Reply:
This is now addressed.